webpack/assets/javascripts/react_app/components/common/table/formatters/translatedCellFormatter.js

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
```

Comment:
There's no React component in this formatter, so you could probably remove this line and use the formatter as a regular function